cmd/cosign/cli/commands.go

```diff
@@ -63,7 +63,7 @@ func New() *cobra.Command {
 				var err error
 				out, err = os.Create(ro.OutputFile)
 				if err != nil {
-					return errors.Wrapf(err, "Error creating output file %s", ro.OutputFile)
+					return fmt.Errorf("error creating output file %s: %w", ro.OutputFile, err)
```

Comment:
also to lowercase for Go idiom, np?

Reply:
Yeah I think we should lowercase the error messages, but otherwise don't make any changes to them.

Reply:
act.